test/config_tests.ml

```diff
@@ -47,7 +59,44 @@ testpass
 remote 10.0.0.1|} in
   Alcotest.(check (result conf_map pmsg)) "basic conf works"
     (Ok minimal_config)
-    (parse_noextern basic)
+    (parse_noextern_client basic)
+
+let minimal_server_config =
+  let open Openvpn.Config in
+    let add_ok_b (b:(b,'a) result) t =
+        Rresult.R.get_ok b |> function B (k,v) -> add k v t
+    in
+  empty
+  |> add Dev (`Tun ,(Some "tun0"))
+  |> add Ping_interval `Not_configured
+  |> add Cipher "AES-256-CBC"
+  |> add Ping_timeout (`Restart 120)
+  |> add Renegotiate_seconds 3600
+  |> add Bind (Some (Some 1195, None))
+  |> add Handshake_window 60
+  |> add Transition_window 3600
+  |> add Tls_timeout 2
+  |> add Resolv_retry `Infinite
+  |> add Auth_retry `None
+  |> add Connect_timeout 120
+  |> add Connect_retry_max `Unlimited
+  |> add Proto (Some `Ipv4, `Tcp (Some `Server))
+  |> add Tls_mode `Server
+  |> add Server ((Ipaddr.V4.of_string_exn "10.89.0.0"), Ipaddr.V4.Prefix.of_string_exn "10.89.0.0/24")
+  |> add_ok_b (a_ca_payload (string_of_file "ca.public.certificate" ) : (b,'a) result)
```

Comment:
Is `ca` required in a minimal config? We don't check for it in the `Config` module.